NEWS.md

```diff
@@ -2,6 +2,12 @@ Hit tab to unselect buttons and scroll through the text using UP/DOWN or
 PGUP/PGDN. All announcements are stored in `/opt/openhabian/docs/CHANGELOG.md`
 for you to lookup.
 
+## Forward web proxy ## September 28, 2021
```

Comment:
Make sure to only keep the 3 most recent announcements in here. We keep all announcements in `doc/CHANGELOG.md` so make sure to add it there.

Reply:
ok